chap-isaref-x86-64.tex

```diff
@@ -239,6 +239,17 @@ \section{Instructions Deprecated in Capability Mode}
   \hline
   A3 & MOV \emph{moffs8},[ER]AX & Move [ER]AX to  (\emph{offset}).\\
   \hline
+  CA & RET \emph{imm16} &
```

Comment:
Hm, deprecated isn't quite the right term for all these, that implies they still exist but aren't recommended

Reply:
Actually, I fixed this more generally in #44.